packages/expo-modules-autolinking/scripts/ios/cocoapods/sandbox.rb

```diff
@@ -35,8 +35,9 @@ class Sandbox
         patched_spec = Specification.from_json(spec_json.to_json)
 
       # Patch podspecs to define module
-      elsif ['ReactCommon', 'React-RCTAppDelegate'].include? name
+      elsif ['ReactCommon', 'React-RCTAppDelegate', 'React-hermes', 'React-jsc'].include? name
```

Comment:
Do we need to publish this change for SDK 47?

Reply:
this is only for testing. because ExpoModulesTestCore as a swift module requires these two pods, we should define them as modules. so we don't have to publish the change.